_episodes/40.md

```diff
@@ -0,0 +1,495 @@
+[Trino 394]({{site.url}}/docs/current/release/release-394.html),
+and
+[Trino 395]({{site.url}}/docs/current/release/release-395.html).
+
+## Concept of the week: Latest features in Apache Iceberg and the Iceberg connector
+
+It has been over a year since we had Ryan on the Trino Community Broadcast as
+guest to discuss what Apache Iceberg is and some detailed usage around the
+connector. Since then, the adoption of Iceberg in our community has skyrocketed.
+Iceberg is truly delivering as much better alternative to the Hive table format. 
+As with anything in the Trino community, we follow the demand and
+needs of the community. 
+
+The initial phase of the Iceberg connector in Trino aimed to provide fast and 
+interoperable read support. A typical usage was Trino alongside other query 
+engines like Apache Spark which supported many of the data modification language
+(DML) SQL features on Iceberg. One of the biggest requests we got as adoption
+increased was the ability to do everything through just Trino. This episode
+dives into some of the latest features that were missing from the early
+iterations of the Iceberg connector, and what has changed in Iceberg as well!
+
+### What is Apache Iceberg?
+
+[Iceberg](https://iceberg.apache.org/) is a next-generation table format that 
+defines a standard around the metadata used to map data to a SQL query engine. 
+It addresses a lot of the maintainability and reliability issues many engineers
+experienced with the way
+[Hive modeled SQL tables]({% post_url 2020-10-20-intro-to-hive-connector %}) 
+over big data files. 
+
+One common confusion to point out is that table format is not equivalent to file
+formats like ORC or Parquet. The table format is the layer that maintains 
+metadata mapping these files to the concept of a table and other common database
+abstractions.
+
+This episode assumes you hsome basic knowledge of Trino and Iceberg already. If
+you are new to Iceberg or need a refresher, we recommend the two older episodes
+about Iceberg and Trino basics:
+
+* [14: Iceberg: March of the Trinos]({{site.url}}/episodes/14.html)
+* [15: Iceberg right ahead!]({{site.url}}/episodes/15.html)
+
+### Why Iceberg over other formats?
+
+Iceberg is one of three of the recent table formats. There is also
+[Delta Lake](https://delta.io) and [Apache Hudi](https://hudi.apache.org), which
+Trino also supports. DeltaLake recently had 
+[a lot of new features open sourced](https://www.databricks.com/blog/2022/06/30/open-sourcing-all-of-delta-lake.html)
+to catch up with Iceberg. While this was a great advancement for the Delta Lake
+community, there are still some missing capabilities that Delta Lake and Hudi 
+have as they remain Hive-like formats by designing the systems to remain 
+backwards compatible actually reintroduces some limitations that existed in
+Hive. 
+
+On top of that, Iceberg has the most support for query engines and puts a heavy 
```

Comment:
```suggestion
Iceberg has the most support for query engines and puts a heavy 
```

still weird ... and probably should change if we drop Delta Lake and Hudi mentions